pkg/storage/apis/obot.obot.ai/v1/mcpservercatalogentry.go

```diff
@@ -17,9 +19,40 @@ type MCPServerCatalogEntry struct {
 	Status MCPServerCatalogEntryStatus `json:"status,omitempty"`
 }
 
+func (in *MCPServerCatalogEntry) GetColumns() [][]string {
+	return [][]string{
+		{"Name", "Name"},
+		{"Tool Reference", "Spec.ToolReferenceName"},
+		{"MCP Catalog", "Spec.MCPCatalogName"},
+		{"Created", "{{ago .CreationTimestamp}}"},
+	}
+}
+
+func (in *MCPServerCatalogEntry) Has(field string) bool {
+	return slices.Contains(in.FieldNames(), field)
+}
+
+func (in *MCPServerCatalogEntry) Get(field string) string {
+	switch field {
+	case "spec.toolReferenceName":
+		return in.Spec.ToolReferenceName
```

Comment:
It looks like this field selector isn't used either. Can it be removed?

Reply:
Same for this one, removing

Reply:
fixed